regulatory/documents/architecture_design_chart.md

```diff
@@ -0,0 +1,11 @@
+---
+id: ADC-001
```

Comment:
Suggestion: perhaps rename these files to contain the document id in the filename, so that it's easier to locate them?